app/Http/Controllers/Api/WaitingRoomsController.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace Gladiator\Http\Controllers\Api;
+
+use Gladiator\Models\WaitingRoom;
+use Illuminate\Http\Request;
+
+class WaitingRoomsController extends ApiController
+{
+    public function calculateSplit(Request $request)
```

Comment:
I think we'd actually want to do a CSRF-style middleware (if anything) here, since this API endpoint is being called from the client-side (and we defs don't want to expose our API key in our HTML templates).